### PR TITLE
Fix a possible NPE

### LINKED ISSUE #3595: export crashes
Since 2020-05-06 every export crashes: 

fehler

### PR DESCRIPTION
**Pull request by matthias-ronge**
Presumably fixes #3595

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java [hunk 1]
```diff
@@ -227,7 +227,8 @@ private ValidationResult checkTheIdentifier(Workpiece workpiece) throws DAOExcep
         User user = ServiceManager.getUserService().getAuthenticatedUser();
         String metadataLanguage = user != null ? user.getMetadataLanguage()
                 : Helper.getRequestParameter("Accept-Language");
-        return LanguageRange.parse(! metadataLanguage.isEmpty() ? metadataLanguage : "en");
+        return LanguageRange
+                .parse(Objects.nonNull(metadataLanguage) && !metadataLanguage.isEmpty() ? metadataLanguage : "en");
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java:231]: I don't think that this change is necessary as `getMetadataLanguage()` is returning a String object in any case. See #3550 for changes in this method.

Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java:231]:
```suggestion
                .parse(StringUtils.isBlank(metadataLanguage) ? metadataLanguage : "en");
```

File: Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java [hunk 2]
```diff
@@ -228,7 +228,7 @@ private ValidationResult checkTheIdentifier(Workpiece workpiece) throws DAOExcep
         String metadataLanguage = user != null ? user.getMetadataLanguage()
                 : Helper.getRequestParameter("Accept-Language");
         return LanguageRange
-                .parse(Objects.nonNull(metadataLanguage) && !metadataLanguage.isEmpty() ? metadataLanguage : "en");
+                .parse(StringUtils.isBlank(metadataLanguage) ? metadataLanguage : "en");
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java:231]: Should `StringUtils.isBlank" call not inverted or the values switched?

Comment (solth) [reply Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java:231]: `StringUtils` isn't even imported, is it? :anguished:
`MetadataValidationService.java:[231,36] cannot find symbol`

Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java:231]: No idea. That was a suggestion from @Kathrin-Huber. I only accepted that on GitHub. Apparently, GitHub doesn't check the class for compile errors.

**Pull request merged**